src/ol/mapbrowserevent.js

```diff
@@ -40,14 +40,14 @@ ol.MapBrowserEvent = function(type, map, browserEvent, opt_dragging,
   this.originalEvent = browserEvent;
 
   /**
-   * The pixel of the original browser event.
+   * The map pixel relative to the viewport corresponding to the original browser event.
    * @type {ol.Pixel}
    * @api stable
    */
   this.pixel = map.getEventPixel(browserEvent);
 
   /**
-   * The coordinate of the original browser event.
+   * The geographical coordinate corresponding the original browser event.
```

Comment:
corresponding *to*? Also, it's not a "geographical coordinate", but the "coordinate in view projection".